alchemy/src/cloudflare/vectorize-index.ts

```diff
@@ -169,7 +169,7 @@ const _VectorizeIndex = Resource(
         if (
           props.adopt &&
           error instanceof CloudflareApiError &&
-          error.message.includes("already exists")
+          error.message.includes("vectorize.index.duplicate_name")
```

Comment:
We should add a test for this case.

Reply:
Pushed a commit with the test, looking good. Also found a bug with delete where we were not idempotent